packages/@react-types/autocomplete/src/index.d.ts

```diff
@@ -14,6 +14,7 @@ import {AsyncLoadable, CollectionBase, LoadingState, SpectrumLabelableProps, Spe
 import {Key} from 'react';
 import {MenuTriggerAction} from '@react-types/combobox';
 import {SpectrumSearchFieldProps} from '@react-types/searchfield';
+
 export interface SearchAutocompleteProps<T> extends CollectionBase<T>, Omit<SpectrumSearchFieldProps, 'onSubmit'> {
```

Comment:
Ah, this seems off now that I look at it more closely. The useSearchAutoComplete aria props extend from SearchAutocompleteProps, but SearchAutocompleteProps extends SpectrumSearchField props which might bring in spectrum specific props that we don't want in those aria props...

I'm happy to approve since this component is in alpha and it was like this prior to this pull, but it should probably extend from AriaSearchFieldProps instead? And maybe even that has too much, I'd need to take a closer look

Reply:
It looks like `SpectrumSearchFieldProps` was used because it imports `SpectrumTextFieldProps` which has icon. Trying something different, but probably should make sure this gets an API review.